test/test_page_without_a_file.rb

```diff
@@ -56,13 +56,17 @@ def render_and_write
           "url"       => "/properties.html",
         }
         attrs.each do |prop, value|
-          # assert the props being accessible in a Jekyll::Page instance
+          # assert a couple of attributes accessible in a regular Jekyll::Page instance
           assert_equal "All the properties.\n", regular_page["content"]
           assert_equal "properties.html", regular_page["name"]
```

Comment:
@ashmaroli Any idea what the purpose of these `regular_page` tests is?

Reply:
Ah.. these should be "*variants*" too.. I'll override your commit.. stay tuned..

Reply:
Never mind.. Its best left as is.. :grin: